dist/profile/manifests/puppetmaster.pp

```diff
@@ -10,6 +10,14 @@
   include ::irc
   include datadog_agent
 
+  # Install ubuntu 20.04 repo package for aarch64
+  # required for running ubuntu 20.04 on arm64 instance
+  # https://puppet.com/docs/pe/2019.8/installing_agents.html
+  # Agent packages can be found on the primary server in
+  # /opt/puppetlabs/server/data/packages/public/<PE VERSION>/
+  class pe_repo::platform::ubuntu_2004_aarch64 {
+  }
```

Comment:
Based on the CI error message (see below), you might want to move the class declaration otuside the parent class block

```text
[2021-07-19T13:02:23.356Z] dist/profile/manifests/puppetmaster.pp - ERROR: pe_repo::platform::ubuntu_2004_aarch64 not in autoload module layout on line 18
[2021-07-19T13:02:23.356Z] dist/profile/manifests/puppetmaster.pp - WARNING: class defined inside a class on line 18
```